app/views/spree/shared/_main_nav_bar.html.erb

```diff
@@ -1,16 +1,28 @@
-<nav class="col-md-12">
-  <div id="main-nav-bar" class="navbar">
-    <ul class="nav navbar-nav" data-hook>
-      <li id="home-link" data-hook><%= link_to Spree.t(:home), spree.root_path %></li>
-    </ul>
-    <ul class="nav navbar-nav navbar-right" data-hook>
-      <li id="link-to-cart" data-hook>
-        <noscript>
-          <%= link_to_cart %>
-        </noscript>
-        &nbsp;
-      </li>
-      <script>Spree.fetch_cart()</script>
-    </ul>
-  </div>
-</nav>
+<!-- <nav class="col-md-12">
+   <div id="main-nav-bar" class="navbar">
+      <ul class="nav navbar-nav" data-hook>
+         <li id="home-link" data-hook><%= link_to Spree.t(:home), spree.root_path %></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right" data-hook>
+         <li id="link-to-cart" data-hook>
+            <noscript>
+               <%= link_to_cart %>
+            </noscript>
+            &nbsp;
+         </li>
+         <script>Spree.fetch_cart()</script>
+      </ul>
+   </div>
+</nav> -->
+<a class="nav-search tablet-visible" id="main-nav-bar"  href="#"><span>Find</span></a>
+<% if spree_current_user.present? %>
+   <%= link_to  destroy_spree_user_session_path, :class => 'nav-profile tablet-hide' do %>
+      <span id="span">Logout </span>
+   <% end %>            
+<% else %>
+   <%= link_to new_spree_user_session_path , :class => 'nav-profile tablet-hide' do %>
+      <span id="span">Login </span>    
+   <% end %>
+<% end %>
+<a class="nav-cart" id="link-to-cart" data-hook href="#"><span>Cart</span><small class="cart-counter">4</small></a> 
```

Comment:
Add link to cart icon and it should redirect to /cart page after click on cart icon